pyaerocom/colocation.py

```diff
@@ -558,6 +558,14 @@ def colocate_gridded_ungridded(gridded_data, ungridded_data, ts_type=None,
         col_freq = str(grid_ts_type)#TS_TYPE_TO_PANDAS_FREQ[grid_ts_type]
         obs_start = start
         obs_stop = stop
+
+
+    latitude = gridded_data.latitude.points
+    longitude = gridded_data.longitude.points
+    lat_range = [np.min(latitude), np.max(latitude)]
+    lon_range = [np.min(longitude), np.max(longitude)]
+    ungridded_data = ungridded_data.filter_by_meta(latitude=lat_range, longitude=lon_range)
+
```

Comment:
@ejgal Isn't this already in `v090dev`? Guess you merged `v090dev` into `flexible_gridded_reader` but not yet into `ReadEMEP`? Can you synchronise both branches with `v090dev` so that I only need to review things that have already been reviewed in PRs for `v090dev`

Reply:
Thought I merged it, but that must have been before these changes.
Both branches are synchronized now.